centreon/tests/e2e/cypress/e2e/Dashboards/01-dashboard-creation.feature

```diff
@@ -1,8 +1,10 @@
-Feature: Creating a new dashboard
+@REQ_MON-18494
+Feature: Manage dashboards (create/delete)
```

Comment:
I prefer the previous Feature names, I suggest we stick to them and update the Jira issue instead. For the deletion, let's create a new one in the existing Epic